src/item/WritableBookBase.php

```diff
@@ -69,29 +69,18 @@ public function getPageText(int $pageId) : string{
 	 *
 	 * @return $this
 	 */
-	public function setPageText(int $pageId, string $pageText) : self{
+	public function setPageText(int $pageId, string $pageText = "") : self{
```

Comment:
This makes no sense. Why would you call something called `setPageText()` and not provide any text ...